imap_processing/decom.py

```diff
@@ -1,5 +1,37 @@
+# Standard
+import logging
+
+from bitstring import ReadError
+
+# Installed
```

Comment:
I still am not a huge fan of these comments in every file, but if you're going to include them then I think this should be above `bitstring` since that is also an extra and not from the standard library.